EventHubs/src/Config/deploy.variables.json

```diff
@@ -1,6 +1,7 @@
 {
   "default": {
-    "ResourceGroupName": "gcm-sumomonitoring-nonprd"
+    "ResourceGroupName": "gcm-sumomonitoring-nonprd",
+    "AppInsightsInstrumentationKey": "892f8766-6c6a-40cd-b435-ea2ebedeea32"
   },
   "prd": {
     "ResourceGroupName": "gcm-sumomonitoring-prd"
```

Comment:
Please include prd AppInsightsInstrumentationKey - prd;

Reply:
Updated :)

Reply:
I think you should also set **68b7cefe-95c6-4ff1-82f3-b96810b89217** as the nonprd value, because we don't need the sumologic specific AppInsights instance anymore, we should use gcm-applicationinsights-(non)prd for everything.